coalib/results/Diff.py

```diff
@@ -244,10 +245,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._original
 
-    @property
-    def modified(self):
+    def _raw_modified(self):
```

Comment:
actually do we still need this property? Maybe we should initially append newlines with `_generate_linebreaks` and consider this the `_original`/`_file` then?